code/modules/mob/mob.dm

```diff
@@ -198,21 +198,29 @@
 			continue
 		if(M in ignored_mobs)
 			continue
-		var/msg = message
+		var/msg = "<span class='notice'>"
```

Comment:
concat is slow in byond, use formatting where ever possible

`var/msg = "<span class='notice'>[message]</span>"`

Reply:
I didn't know it could be that easy, thanks!